leshan-server-demo/src/main/java/org/eclipse/leshan/server/demo/LeshanServerDemo.java

```diff
@@ -541,19 +546,41 @@ public static void createAndStartServer(String webAddress, int webPort, String l
             jettyAddr = new InetSocketAddress(webAddress, webPort);
         }
         Server server = new Server(jettyAddr);
-        WebAppContext root = new WebAppContext();
-        root.setContextPath("/");
-        root.setResourceBase(LeshanServerDemo.class.getClassLoader().getResource("webapp").toExternalForm());
-        root.setParentLoaderPriority(true);
+        /*
+         * WebAppContext root = new WebAppContext(); root.setContextPath("/");
+         * root.setResourceBase(LeshanServerDemo.class.getClassLoader().getResource("webapp").toExternalForm());
+         * root.setParentLoaderPriority(true); server.setHandler(root);
+         */
```

Comment:
Commented code

Reply:
This code should be added again when we will remove old version of demo.
So I will keep it (I will add a comment to explain)